crates/primitives/src/chain/spec.rs

```diff
@@ -157,6 +157,49 @@ pub static SEPOLIA: Lazy<Arc<ChainSpec>> = Lazy::new(|| {
     .into()
 });
 
+/// The Holesky spec
+pub static HOLESKY: Lazy<Arc<ChainSpec>> = Lazy::new(|| {
+    ChainSpec {
+        // TODO: <https://github.com/gakonst/ethers-rs/pull/2569>
+        chain: Chain::Id(17000),
+        genesis: serde_json::from_str(include_str!("../../res/genesis/holesky.json"))
+            .expect("Can't deserialize Holesky genesis json"),
+        genesis_hash: Some(H256(hex!(
+            "ff9006519a8ce843ac9c28549d24211420b546e12ce2d170c77a8cca7964f23d"
+        ))),
+        paris_block_and_final_difficulty: Some((0, U256::from(1))),
+        fork_timestamps: ForkTimestamps::default().shanghai(1694790240),
+        hardforks: BTreeMap::from([
+            (Hardfork::Frontier, ForkCondition::Block(0)),
+            (Hardfork::Homestead, ForkCondition::Block(0)),
+            (Hardfork::Dao, ForkCondition::Block(0)),
+            (Hardfork::Tangerine, ForkCondition::Block(0)),
+            (Hardfork::SpuriousDragon, ForkCondition::Block(0)),
+            (Hardfork::Byzantium, ForkCondition::Block(0)),
+            (Hardfork::Constantinople, ForkCondition::Block(0)),
+            (Hardfork::Petersburg, ForkCondition::Block(0)),
+            (Hardfork::Istanbul, ForkCondition::Block(0)),
+            (Hardfork::MuirGlacier, ForkCondition::Block(0)),
+            (Hardfork::Berlin, ForkCondition::Block(0)),
+            (Hardfork::London, ForkCondition::Block(0)),
+            (
+                Hardfork::Paris,
+                ForkCondition::TTD { fork_block: None, total_difficulty: U256::ZERO },
```

Comment:
note: add a test that checks that paris hardfork is activated correctly at genesis